src/ui/public/stringify/types/boolean.js

```diff
@@ -0,0 +1,41 @@
+import IndexPatternsFieldFormatProvider from 'ui/index_patterns/_field_format/field_format';
+
+export default function TruncateFormatProvider(Private) {
+
+  let FieldFormat = Private(IndexPatternsFieldFormatProvider);
+
+  class Bool extends FieldFormat {
+
+    constructor(params) {
+      super(params);
+    }
+
+    _convert(value) {
+
+      if (typeof value === 'string') {
+        value = value.trim().toLowerCase();
+      }
+
+      switch (value) {
+        case false:
+        case 0:
+        case 'false':
+        case 'no':
+          return 'false';
+        case true:
+        case 1:
+        case 'true':
+        case 'yes':
+          return 'true';
+        default:
+          return value.toString();
+      }
+    }
+  }
+
+  Bool.id = 'boolean';
+  Bool.title = 'Boolean';
+  Bool.fieldType = ['boolean'];
```

Comment:
OK. That makes sense if people want to display a value as boolean, but not necessarily map it as such in their index.